### PR TITLE
Added a `master.toml` file

### PR DESCRIPTION
**Pull request by etorreborre**
This file is used by the Ambiata's private `master` project to build the project on the internal CI environment.

This PR relies on `ambiata/master-ambiata#10`.

@charleso @nhibberd

### CODE REVIEW

**Comment by markhibberd** [comment]
We should oss the runners and master.

**Comment by etorreborre** [comment]
I thought that this would be the next step but maybe we need to move more projects to `master` first. I was also proposing to remove the [`sha` requirement](https://github.com/ambiata/master/issues/32) and use unique version numbers instead to do the caching but it might be too Ambiata-centric then?

File: master.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[master]
+   version = 1
+   runner = "s3://ambiata-dist/master-ambiata/scala-oss-$VERSION"
```
Comment (charleso) [inline master.toml]: Just to confirm - the `VERSION` here should be hard-coded

Comment (etorreborre) [inline master.toml]: Yes, I am just waiting on the `master-ambiata` PR to be merged first.

**Pull request merged**